hail/python/hailtop/utils/utils.py

```diff
@@ -116,7 +116,7 @@ def grouped(n: int, ls: List[Any]) -> Iterable[Any]:
         yield group
 
 
-def partition(k, ls):
+def partition(k: int, ls: List[Any]) -> Iterable[Any]:
```

Comment:
The return type should be `Iterable[List[Any]]`. Can you tell me why?

Reply:
It's because `partition` returns a list with either `int` or `str`.